cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/CseListener.java

```diff
@@ -38,6 +43,7 @@ public class CseListener implements MessageListener {
     private final JsonApiConverter jsonApiConverter;
     private final CseRunner cseServer;
     private final Logger businessLogger;
+    private Map<String, CompletableFuture<CseResponse>> futurs = new HashMap<>();
```

Comment:
you can make it final and call it `futures`

Reply:
renamed